plugins/services/src/js/components/ServiceConfigEnvironmentVariablesSectionDisplay.js

```diff
@@ -68,7 +78,7 @@ class ServiceConfigEnvironmentVariablesSectionDisplay extends ServiceConfigBaseS
             return (
               <Table
                 key="secrets-table"
-                className="table table-simple table-break-word flush-bottom"
+                className="table table-simple table-align-top table-break-word table-fixed-layout flush-bottom"
```

Comment:
yes, we should use this on all of these tables, so the edit link (and other variables) will stick to the top, so we don't end up in this situation:
![](https://cl.ly/2w460x1E0345/Image%202016-12-14%20at%2010.46.53.png)

But get this instead:
![](https://cl.ly/160n2w0d373y/Image%202016-12-14%20at%2010.47.27.png)